app-ng/src/utilities.lisp

```diff
@@ -4,36 +4,27 @@
 
 (in-package #:qvm-app-ng)
 
-(defun session-info ()
-  ;; Stub implementation for FORMAT-LOG, below. See app/src/utilities.lisp for the original.
-  "")
+(defun get-random-state (arg)
+  (etypecase arg
+    (null (qvm:seeded-random-state nil))
+    (unsigned-byte (qvm:seeded-random-state arg))))
+
+(defun hex-char-p (char &key (case-sensitive-p t))
```

Comment:
do away with `case-sensitive-p` and just make it insensitive by default. Why not?

Reply:
Because I canonicalize to lowercase, and this gets called by the `VALID-PERSISTENT-QVM-TOKEN-P` to make sure the input is both valid and canonicalized to lowercase. This function was originally case-insensitive and I made the test on the persistent qvm hash-table `'eqaulp` rather than `equal`.  After some back and forth with notmgsk, I decided that I would accept either upper or lowercase as valid input, but then canonicalize to lowercase to have a consistent internal format... I could always switch back or just have the validator ignore case, etc., but it seemed reasonable to let the caller decide if they want case-sensitivity or not.